src/Symfony/Component/Security/Http/EventListener/CheckRememberMeConditionsListener.php

```diff
@@ -69,4 +70,28 @@ public static function getSubscribedEvents(): array
     {
         return [LoginSuccessEvent::class => ['onSuccessfulLogin', -32]];
     }
+
+    private function getParameter(Request $request, string $parameterName): mixed
+    {
+        $parameter = ParameterBagUtils::getRequestParameterValue($request, $parameterName);
+        if (null !== $parameter) {
+            return $parameter;
+        }
+
+        return $this->getJsonBodyParameter($request, $parameterName);
+    }
+
+    private function getJsonBodyParameter(Request $request, string $parameterName): mixed
+    {
+        if ('application/json' === $request->headers->get('Content-Type')) {
+            $data = json_decode($request->getContent());
+            if (!$data instanceof \stdClass) {
+                return null;
+            }
+
+            return $data->{$parameterName} ?? null;
+        }
+
+        return null;
+    }
```

Comment:
Why do not add this logic directly into the ParameterBagUtils::getRequestParameterValue function?

Reply:
I‘m not sure if this wouldn‘t have any unwanted side effects since `ParameterBagUtils::getRequestParameterValue` is called from other functions. Adding the code that way would allow others values like e.g. CSRF tokens to be submitted via JSON in the request‘s body as well.

Reply:
Could you please merge the two added private helpers in one method?

Reply:
> Could you please merge the two added private helpers in one method?

done